backend/transcribee_backend/api/views.py

```diff
@@ -23,7 +23,7 @@
 
 class DocumentViewSet(viewsets.ModelViewSet):
     def get_queryset(self):
-        return Document.objects.filter(user=self.request.user)
+        return Document.objects.filter(user=self.request.user).order_by("-created_at")
```

Comment:
I think `-changed_at` would be the better one here, so the user always sees the document it just edited first.

Reply:
Also I think we might want to add an ordering to the document model, so we always get that ordering? `created_at` might be the right one there, so we have a more permanent sorting https://docs.djangoproject.com/en/4.1/ref/models/options/#ordering

in base/models.py:
```
class Document(models.Model):
    class Meta:
        ordering = ["-created_at", "id"]
```